### ISSUE TITLE
Latin language

### ISSUE THREAD

**Issue by JEFE01** [paulpacifico/shutter-encoder#206]
File for conversion to Latin language
[latin language.zip](https://github.com/user-attachments/files/16077408/latin.language.zip)

**Comment by paulpacifico** [comment]
Hello, thanks a lot for your translation! I will add it for the next release ;-)
Do you have a PayPal account to send you some money for you work?

Best,
Paul.

**Comment by JEFE01** [comment]
of course, I have already sent you the information to the email received, thank you.

**Comment by JEFE01** [comment]
yes of course, user: @jefemex01
mail: jefemex01@gmail.com

**Comment by paulpacifico** [comment]
Thanks !! 🙏

**Comment by JEFE01** [comment]
updated language

[latin.zip](https://github.com/user-attachments/files/22075434/latin.zip)

**Comment by paulpacifico** [comment]
Added for the next release ;-)

**Comment by JEFE01** [comment]
change of the la (Latin) file from version to Spanish (disappears) by es_LAT for a better search of the language and some corrections.
Thank you...
[es_LAT.zip](https://github.com/user-attachments/files/22188867/es_LAT.zip)

**Comment by JEFE01** [comment]
because if the file is changed to es_LAT the program does not take it, since it takes the first version to Spanish I return it to (la), since it does not take the language

[la.zip](https://github.com/user-attachments/files/22238014/la.zip)

**Comment by paulpacifico** [comment]
Perfect! Everything was updated except for

lblVideoFadeIn=Fundido de entrada de vídeo:
lblVideoFadeOut=Fundido de salida de vídeo:
lblAudioFadeIn=Fundido de entrada de audio:
lblAudioFadeOut=Fundido de salida de audio:

Because it does not fit in the UI, do you have a shorter suggestion?
Best,
Paul.

**Comment by JEFE01** [comment]
correction to the above points, thank you...

[la.zip](https://github.com/user-attachments/files/22541407/la.zip)
